src/games-spa/src/components/Word/styles.ts

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+import Button from '@material-ui/core/Button';
+
+type Card = {
+  cardType: number;
+};
+
+export const GameCardContainer = styled.div`
+  display: block;
+  width: 100%;
+  height: 100%;
+`;
+
+export const GameCard = styled(Button)``;
+
+export const RevealedGameCard: any = styled.div<Card>`
+  border-radius: 4px;
+  text-align: center;
+  padding: 8px 16px;
```

Comment:
Mind using the material ui spacing here instead of having the padding as css?

Reply:
This is not a material ui component so without some work I don't now that it has access to the default spacing. I've also never tried, so I'll look into that more. Or if you know how I am totally open to suggestions!

Reply:
I suppose I could change it from a div to the Box mui component. I'll give that a try tonight but in theory that should adjust for both the styled-component theme values and mui theme values.